WRITING.md

```diff
@@ -71,6 +73,75 @@ Using `<blockquote>` for notes is deprecated.
 
 You can see all three currently-used styles (we don't have any "danger" yet!) in action on the [Astro Components Page](https://docs.astro.build/en/core-concepts/astro-components/).
 
+### Badge Component
+
+Sometimes it is helpful to add a small badge to some content to label or highlight it. You can use the `<Badge />` component for this.
+
+Badges work best when they only contain a single word or — at a push — two words. Think of them as a tag or label for something, not a way to highlight longer passages of text.
+
+```md
+---
+setup: |
+  import Badge from '~/components/Badge.astro';
+---
+
+<Badge>Nice</Badge>
+```
+
+By default, the badge uses a muted colour scheme to blend in. It also has an accented variant that can be used if you need it to stand out more from the surrounding context:
+
+```md
+<Badge variant="accent">Wow!</Badge>
+```
+
+#### Examples
+
+- The [Editor Setup](https://docs.astro.build/en/editor-setup/#other-code-editors) page uses `<Badge />` to distinguish community and official editor support.
+- The [Deployment guides navigation component](https://docs.astro.build/en/guides/deploy/) uses `<Badge />` internally to label the deployment types each service supports.
+
+### Since Component
+
+As features are added to Astro, it can be helpful to document _when_ they were added. This allows users to easily see if the version of Astro they are running supports a specific feature as described in the docs.
+
+You can use the `<Since />` component to display this information in a standardised way. This component takes a single `v` prop, which indicates the Astro version the feature was added in.
```

Comment:
```suggestion
You can use the `<Since />` component to display this information in a standardized way. This component takes a single `v` prop, which indicates the version of Astro in which the feature was added.
```
Yeah, the "in which" is a judgment call to be correct vs common. I'd do this personally, but I won't object to ending the sentence with a proposition if you feel it sounds better.

BUT, I did explicitly choose "version of Astro" to get the "v is for version" connection more closely aligned.